static/src/stylesheets/module/crosswords/_vars.scss

```diff
@@ -0,0 +1,14 @@
+// based on cellSize + borderSize from
+// projects/common/modules/crosswords/constants.js
+$xword-cell-width: 31px;
+$xword-border-width: 1px;
+
+$xword-highlight: lighten(colour(crosswordAccent2), 30%);
+$xword-focussed: lighten(colour(crosswordAccent2), 10%);
+$xword-ink: #3f4ac3; // biro blue
+
+$xword-grid-sizes: (
```

Comment:
yeah appears to be. if not, we'll need to create some mad helper classes, but has been so far